releasenotes/notes/fix-linear-solvers-api-2b4df93a42263337.yaml

```diff
@@ -0,0 +1,9 @@
+---
+prelude: >
+    The linear solvers package API documentation was incomplete.
```

Comment:
This only concerns the final releasenote for the release, it's not for individual PRs 🙂 
```suggestion
```